frontend/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
```

Comment:
Don't push .git ignore file.

Reply:
Both the folders are different and require different package.json